src/main/kotlin/no/nav/familie/oppdrag/service/AvstemmingService.kt

```diff
@@ -18,17 +20,42 @@ class AvstemmingService(
         val avstemmingMapper = AvstemmingMapper(oppdragSomSkalAvstemmes, fagsystem)
         val meldinger = avstemmingMapper.lagAvstemmingsmeldinger()
 
+        if (meldinger.isEmpty()) {
+            LOG.info("Ingen oppdrag å gjennomføre grensesnittavstemming for.")
+            return
+        }
+
         LOG.info("Utfører grensesnittavstemming for id: ${avstemmingMapper.avstemmingId}, ${meldinger.size} antall meldinger.")
 
         meldinger.forEach {
             avstemmingSender.sendGrensesnittAvstemming(it)
         }
 
         LOG.info("Fullført grensesnittavstemming for id: ${avstemmingMapper.avstemmingId}")
+
+        oppdaterMetrikker(fagsystem, meldinger[1].grunnlag)
+    }
+
+    fun oppdaterMetrikker(fagsystem: String, grunnlag: Grunnlagsdata) {
+        Metrics.counter("familie.oppdrag.grensesnittavstemming","fagsystem", fagsystem, "status", Status.GODKJENT.status,
+                "beskrivelse", Status.GODKJENT.beskrivelse).increment(grunnlag.godkjentAntall.toDouble())
+        Metrics.counter("familie.oppdrag.grensesnittavstemming","fagsystem", fagsystem, "avvist", Status.AVVIST.status,
+                "beskrivelse", Status.AVVIST.beskrivelse).increment(grunnlag.avvistAntall.toDouble())
+        Metrics.counter("familie.oppdrag.grensesnittavstemming","fagsystem", fagsystem, "mangler", Status.MANGLER.status,
+                "beskrivelse", Status.MANGLER.beskrivelse).increment(grunnlag.manglerAntall.toDouble())
+        Metrics.counter("familie.oppdrag.grensesnittavstemming","fagsystem", fagsystem, "varsel", Status.VARSEL.status,
+                "beskrivelse", Status.VARSEL.beskrivelse).increment(grunnlag.varselAntall.toDouble())
```

Comment:
Kan taggen på alle disse hete "status"? Ser det gjør det på den første :) Det går også an å lagre counterne i variabler/et map på klassenivå, så slipper du å kalle counter()-metoden på nytt hver gang du skal oppdatere metrikkene. Er riktignok usikker på om det gjør noen forskjell mtp performance eller minne...

Reply:
Godt observert! Det skulle stå status for alle 4 ja :) Hmm, såvidt jeg ser fra andre eksempler må man kalle counter()-metoden hver gang.